internal/provider/resource_aci_out_of_band_contract_test.go

```diff
@@ -279,23 +274,33 @@ resource "aci_out_of_band_contract" "allow_test_2" {
 `
 
 const testConfigVzOOBBrCPMin = `
+data "aci_system" "version" {
+  pod_id    = "1"
+  system_id = "1"
+}
+
 resource "aci_out_of_band_contract" "test" {
   name = "test_name"
 }
 `
 
 const testConfigVzOOBBrCPAll = `
+data "aci_system" "version" {
+  pod_id    = "1"
+  system_id = "1"
+}
+
 resource "aci_out_of_band_contract" "test" {
   name = "test_name"
   annotation = "annotation"
   description = "description_1"
-  intent = "estimate_add"
```

Comment:
Why are we removing these?

Reply:
Introduced version_mismatch in default which takes care of this.